server.js

```diff
@@ -28,7 +28,7 @@ app.all('/api/*', (req, res) => {
 
   axios(options)
     .then((results) => {
-      res.send(results.data);
+      res.status(results.status).send(results.data);
     })
     .catch((err) => {
       console.log(err);
```

Comment:
We should also probably send back the error incase. I noticed I was getting success messages, but when I inspected it in the console the status was actually 404